### ISSUE TITLE
currently unable to boot strap compiler or compile anything else

### ISSUE THREAD

**Issue by fuzzy** [vlang/v#720]
**V version:** v0.1.3
**OS:** Linux kerplop 5.1.8-1-MANJARO #1 SMP PREEMPT Sun Jun 9 20:44:14 UTC 2019 x86_64 GNU/Linux
**C compiler version:** gcc version 8.3.0 (GCC)

**What did you do?**
Try to bootstrap the compiler

**What did you expect to see?**
successful return to shell prompt
 
**What did you see instead?**
The following is a direct paste. Step by step I bootstrapped the c version, compiling the second stage fails. The same error is present if I try to build any of the examples, I have included that below this first block.
```
[master][~/work/build/v/compiler]$ gcc -w -o v.stage1 v.c    
*[master][~/work/build/v/compiler]$ ./v.stage1 -o v.stage2 .
!!! +
!!! -
pass=2 fn=`utf32_to_str_no_malloc`
panic: utf8.v:287
Fn "tos" wrong arg #1. Expected "byte*" (s)  but got "byteptr"
*[master][~/work/build/v/compiler]$ ./v.stage1 version      
V 0.1.3
```

```
*[master][~/work/build/v/examples]$ ../compiler/v.stage1 log.v
pass=2 fn=`utf32_to_str_no_malloc`
panic: utf8.v:287
Fn "tos" wrong arg #1. Expected "byte*" (s)  but got "byteptr"
```

**Comment by medvednikov** [comment]
I just tried

```
git clone https://github.com/vlang/v
cd v/compiler
make
```

And it works fine.

Try `rm ~/.vlang/VROOT`

**Comment by fuzzy** [comment]
indeed it does work fine! Closing this one out, but taking the opportunity to say that I am pretty impressed so far, I found V yesterday and I'm *REALLY* looking forward to the c generation as I use a lot of ARM systems and that would enable me to target them with a single extra step of code transpiling. As a workaround until *hopefully* native machine code generation supports arm(64), I find that workable.
